homeassistant/components/telegram_bot/__init__.py

```diff
@@ -782,7 +782,14 @@ def process_message(self, data):
             if event_data is None:
                 return message_ok
 
-            event_data[ATTR_DATA] = data[ATTR_DATA]
+            query_data = data[ATTR_DATA]
+            event_data[ATTR_DATA] = query_data
```

Comment:
Simplify the above two lines:

```python
query_data = event_data[ATTR_DATA] = data[ATTR_DATA]
```

Reply:
I usually prefer to keep things on separate lines to reduce cognitive load. Is this generally a common convention?

Reply:
I think it's easier to read personally.